### PR TITLE
Version 2

### PR DESCRIPTION
**Pull request by chadicus**
#### What does this PR do?                                                                                              
- [x] Update LICENSE file
- [x] Change package owner to `traderinteractive`
- [x] Change Namespace to `TraderInteractive`
- [x] Add .github files  CODEOWNERS, PULL_REQUEST_TEMPLATE, ISSUE_TEMPLATE & CONTRIBUTING
- [x] Remove support for PHP < 7
- [x] Switch to PSR2 coding standard
- [x] Update README
- [x] Remove composer.lock from version control
- [x] Update Scrutinizer config

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,11 +16,11 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "~5.6 || ~7.0",
+        "php": "^7.0",
         "jenssegers/chef": "dev-master",
         "nubs/hiatus": "~1.2"
```
Comment (hotline-emu) [inline composer.json]: `^1.2`?

File: composer.json [hunk 2]
```diff
@@ -16,11 +16,11 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "~5.6 || ~7.0",
+        "php": "^7.0",
         "jenssegers/chef": "dev-master",
```
Comment (hotline-emu) [inline composer.json]: Do you have to do dev-master?

File: src/Chef.php [hunk 3]
```diff
@@ -32,9 +33,9 @@ public function __construct(ChefApi $chef)
     public function patchDatabag($name, $item, array $data)
```
Comment (hotline-emu) [inline src/Chef.php]: Can you type any of these?

File: src/Chef.php [hunk 4]
```diff
@@ -47,14 +48,19 @@ public function patchDatabag($name, $item, array $data)
     public function getNodes($role = null)
```
Comment (hotline-emu) [inline src/Chef.php]: Can you type this?

File: src/Chef.php [hunk 5]
```diff
@@ -66,6 +72,6 @@ public function getNodes($role = null)
      */
     public function deleteNode($node)
```
Comment (hotline-emu) [inline src/Chef.php]: Type?

File: src/ChefEc2.php [hunk 6]
```diff
@@ -1,36 +1,39 @@
...
-     * @param string $chefServerUrl The url for the chef API.
-     * @param array $credentials The credentials needed to interact with knife-ec2.  Includes awsAccessKeyId and awsSecretAccessKey.
+     * @param string $baseKnifeCommand The base command for knife.  could be 'knife' if knife is already in the path.
+     *                                 If using bundler to install knife-ec2, you could do something
+     *                                 like 'BUNDLE_GEMFILE=/path/to/Gemfile bundle exec knife'.
+     * @param string $chefServerUrl    The url for the chef API.
+     * @param array  $credentials      The credentials needed to interact with knife-ec2.  Includes awsAccessKeyId and
+     *                                 awsSecretAccessKey.
      */
     public function __construct($baseKnifeCommand, $chefServerUrl, array $credentials)
```
Comment (hotline-emu) [inline src/ChefEc2.php]: Types?

File: src/ChefEc2.php [hunk 7]
```diff
@@ -87,33 +90,38 @@ public function createServer(
      * @param string $query The chef query to specify what servers to update.
      * @param string $progressFile The filename to send the knife output to.
      * @param array $options Additional options for knife ssh.  For example: ['-groups' => 'foo']
-     * @param array $chefOptions Additional options for chef client.  For example: ['--override-runlist' => 'role[foo]']
+     * @param array $chefOptions Additional options for chef client. For example: ['--override-runlist' => 'role[foo]']
      * @return void
      */
     public function updateServers($query, $progressFile = null, array $options = [], array $chefOptions = [])
```
Comment (hotline-emu) [inline src/ChefEc2.php]: Types?

File: composer.json (hunk 2 above)
Comment (chadicus) [reply composer.json]: yes

**Pull request merged**